pkg/rootless/rootless_linux.go

```diff
@@ -345,6 +347,31 @@ func joinUserAndMountNS(pid uint, pausePid string) (bool, int, error) {
 	return true, int(ret), nil
 }
 
+func getConfiguredMappings() ([]idtools.IDMap, []idtools.IDMap, error) {
+	var uids, gids []idtools.IDMap
+	username := os.Getenv("USER")
```

Comment:
The getusername block below, has been implemented elsewhere, I believe and should be used or should be put into a separate function.

Reply:
I've added a patch to make the new function in rootless public so it can be used from pkg/util